styleguide/screens/Patterns/Tabs/Tabs.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, C } from '../../../components/Scaffold/Scaffold';
+
+import Tabs from '../../../../components/Tabs/Tabs';
+import Tab from '../../../../components/Tabs/Tab';
+
+import m from '../../../../globals/modifiers.css';
+
+const TabsDocumentation = () => (
+  <div>
+    <H level={ 1 }>Loader</H>
+    <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+      <C>Tabs</C> provide an easy way to navigate between views on a single page,
+      where content is related. For cases when you need page level tabs, <C>TabBar</C>.
+    </T>
+    <D>
+      <Specimen
+        classNames={ {
+          specimenContainer: m.par,
+        } }
+        code={ dedent`
+          <Tabs>
+            <Tab label="One">
+              One
```

Comment:
Could we make it more clear that the "One" and "Two" get rendered as the body of the tab?

Reply:
I could change the tab content to "body one" and "body two"?

Reply:
Sure. Can see how that looks 👍